tasks/main.yml

```diff
@@ -57,8 +57,11 @@
 - name: Ensure Sonar is running and set to start on boot.
   service: name=sonar state=started enabled=yes
 
-- name: Allow Sonar time to build on first start.
-  pause: seconds=180
+- name: Ensure Sonar is started.
+  wait_for:
+    path: /usr/local/sonar/logs/sonar.log
```

Comment:
The only thing I wonder—is it possible this log path is different by distro, or in different versions?